src/Sparrow/Json/JsonOperationContext.cs

```diff
@@ -923,10 +923,19 @@ protected internal virtual void Renew()
             if (Disposed)
                 ThrowObjectDisposed();
 
-            if (_perCorePathCache.TryPull(out _activeAllocatePathCaches) == false)
-                _activeAllocatePathCaches = new PathCache();
-            if (_perCoreLazyStringValuesList.TryPull(out _allocateStringValues) == false)
-                _allocateStringValues = new FastList<LazyStringValue>(256);
+            if (_activeAllocatePathCaches == null)
+            {
+                if (_perCorePathCache.TryPull(out _activeAllocatePathCaches) == false)
+                    _activeAllocatePathCaches = new PathCache();
+            }
```

Comment:
If `_activeAllocatePathCaches  != null` then don't we need to clean. We're gonna reuse the existing instance.

Reply:
After talking to Pawel now I realized my comment isn't relevant